app/main.py

```diff
@@ -46,9 +44,9 @@ def check_word_in_tweet(word, data):
 
 
 
-ds_tweets['google'] = check_word_in_tweet('#google', ds_tweets)
+ds_tweets['google'] = sentiment.get_hashtag_data()['#google']
 
-ds_tweets['apple'] = check_word_in_tweet('#apple', ds_tweets)
+ds_tweets['apple'] = sentiment.get_hashtag_data()['#apple']
```

Comment:
As well

Reply:
For check word in tweets should i call the function from sentimental analysis? or have it on main? my plan was to put it inside sentimental analysis.py and call it from there

Reply:
I guess import the whole class and instantiate it in main. then call check_word_ function

You could do it like this

```
from sentiment_analysis import SentimentAnalysis

sentiment = SentimentAnalysis(ds_tweets=get_tweets())

sentiment.set_hashtag_data('5T','#apple','#google') 

check_fun = sentiment.check_word_in_tweet 

// then just replace get_hashtag with function call of check_fun() at all places
```

Then fianlly remove the instantiation in sentiment_analysis.py

Reply:
Also, it would be nice to rename the class to SentimentAnalysis instead of SentimentalAnalysis

Reply:
Named it to SentimentAnalysis and instantiated it inside main